packages/react/src/patterns/sub-patterns/LinkList/__stories__/LinkList.stories.js

```diff
@@ -0,0 +1,50 @@
+import './index.scss';
+import { select, text, withKnobs } from '@storybook/addon-knobs';
+import LinkList from '../LinkList';
+import React from 'react';
+import readme from '../README.md';
+import { settings } from 'carbon-components';
+import { storiesOf } from '@storybook/react';
+
+const { prefix } = settings;
+
+storiesOf('Patterns (Sub-Patterns)|LinkList', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const heading = text('heading(required):', 'Tutorials');
+    const types = ['local', 'external'];
+    const type = [
+      select('cta 1 type', types, types[0]),
+      select('cta 2 type', types, types[0]),
```

Comment:
I think we should give them more options to stress test than just giving them types to choose from. They can do that in the `CTA` story.